### PR TITLE
[DOC-BUG] Fixes broken link in Graph commands overview

### PR DESCRIPTION
**Pull request by nermiller**
Closes https://redislabs.atlassian.net/browse/DXDOCS-281

### CODE REVIEW

File: docs/docs/commands.md [hunk 1]
```diff
@@ -14,11 +14,11 @@ RedisGraph exposes graph database functionality within Redis using the [openCyph
 
 ### RedisGraph API
 
-Command details can be retrieved by filtering for the [module](/commands/?group=graph) or for a specific command, e.g. [`GRAPH.QUERY`](/commands/?name=graph.query).
+Command details can be retrieved by filtering for the [module](/commands/?group=graph) or for a specific command, e.g., `GRAPH.QUERY`.
 The details include the syntax for the commands, where:
 
-*   Command and subcommand names are in uppercase, for example [`GRAPH.CONFIG`](/commands/?name=graph.config) or `SET`
-*   Optional arguments are enclosed in square brackets, for example `[timeout]`
+*   Command and subcommand names are in uppercase, for example, `GRAPH.CONFIG SET`.
```
Comment (LiorKogan) [inline docs/docs/commands.md:20]: I don't think this is correct. I think it is case agnostic.

```
gRaph.QuEry g "return 0"
1) 1) "0"
2) 1) 1) (integer) 0
3) 1) "Cached execution: 0"
   2) "Query internal execution time: 1.998600 milliseconds"
```

**Review by LiorKogan — CHANGES_REQUESTED** [review]

File: docs/docs/commands.md (hunk 1 above)
Comment (nermiller) [inline docs/docs/commands.md:20]:
```suggestion

```

**Review by LiorKogan — APPROVED** [review]

**Comment by nermiller** [comment]
@LiorKogan mind merging it as well? Thank you so much!

**Comment by LiorKogan** [comment]
@nermiller sorry I don't have permission.
@swilly22 can you please merge it?

**Pull request merged**